setup.py

```diff
@@ -57,19 +60,19 @@ def run(self):
         'click==8.1.3',
         'anchorpy[cli]==0.9.1',
         'asyncclick==8.1.3.2',
-        'prettytable=3.4.1'
```

Comment:
This one was a major issue (typo). The rest is more formatting stuff.

Reply:
Huh! That's where this dep problem came from the other day when i tried to use this lib in auto deployment... Thanks for fixing it!